### DISCUSSION TITLE
LC0033

### DISCUSSION THREAD

**Discussion by SorenBogelund-ECIT** [StefanMaron/BusinessCentral.LinterCop#679] [Q&A]
Thank you developing this great LinterCop!

Why do I get a warning about the runtime version falling behind, when I haven't specified any runtime version in app.json?
![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/169046245/bfb6f39b-89f7-4ad7-96e8-4a274b7d7661)

**Comment by Arthurvdv** [comment]
https://github.com/StefanMaron/BusinessCentral.LinterCop/issues/683
